SignInCheckIn/SignInCheckIn/Controllers/EventController.cs

```diff
@@ -99,6 +99,24 @@ public IHttpActionResult GetEventRoomAgesAndGrades([FromUri] int eventId, [FromU
             }
         }
 
+        [HttpGet]
+        [ResponseType(typeof(EventRoomDto))]
+        [VersionedRoute(template: "events/{eventId:int}/groups/unassigned", minimumVersion: "1.0.0")]
+        [Route("events/{eventId:int}/groups/unassigned")]
+        public IHttpActionResult GetEventUnassignedGroups([FromUri] int eventId)
+        {
+            try
+            {
+                return Authorized(token => Ok(_roomService.GetEventUnassignedGroups(token, eventId)),
+                                  () => Ok(_roomService.GetEventUnassignedGroups(null, eventId)));
```

Comment:
What happens if they don't have a token here? I would think we just want this to happen if they have a token.